src/main/kotlin/org/coralprotocol/coralserver/orchestrator/runtime/Docker.kt

```diff
@@ -0,0 +1,153 @@
+package org.coralprotocol.coralserver.orchestrator.runtime
+
+import com.chrynan.uri.core.Uri
+import com.github.dockerjava.api.async.ResultCallback
+import com.github.dockerjava.api.exception.NotModifiedException
+import com.github.dockerjava.api.model.Frame
+import com.github.dockerjava.api.model.StreamType
+import com.github.dockerjava.core.DefaultDockerClientConfig
+import com.github.dockerjava.core.DockerClientBuilder
+import com.github.dockerjava.core.DockerClientConfig
+import io.github.oshai.kotlinlogging.KotlinLogging
+import kotlinx.coroutines.withContext
+import kotlinx.coroutines.withTimeoutOrNull
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import org.coralprotocol.coralserver.orchestrator.ConfigValue
+import org.coralprotocol.coralserver.orchestrator.OrchestratorHandle
+import org.coralprotocol.coralserver.orchestrator.runtime.executable.EnvVar
+import kotlin.time.Duration.Companion.seconds
+
+private val logger = KotlinLogging.logger {}
+
+@Serializable
+@SerialName("docker")
+data class Docker(
+    val container: String,
```

Comment:
Can we rename `container` to `image` here? More inline with Docker jargon
```suggestion
    val container: String,
```
```suggestion
    val image: String,
```